### PR TITLE
docs: updated Readme and cli-docs generation

### PR DESCRIPTION
**Pull request by olensmar**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,54 +29,39 @@ Known vulnerabilities: [![Testkube](https://snyk.io/test/github/kubeshop/testkub
...
-<p align="center">
-    <img src="assets/testkube-intro.gif">
-</p>
-
-Testkube natively integrates test orchestration and execution into Kubernetes and your CI/CD/GitOps pipeline. It decouples test artifacts and execution from CI/CD tooling; tests are meant to be part of your clusters state and can be executed as needed:
-- Kubectl plugin
-- Externally triggered via API (CI, external tooling, etc)
-- Automatically on deployment of annotated/labeled services/pods/etc (WIP)
+1. **Define** - Use Test Workflows to configure executions of your current testing tools or scripts. 
+  Orchestrate multiple Workflows to build complex Suites for System Testing - [Read More](https://docs.testkube.io/articles/defining-tests)
```
Comment (jfermi) [inline README.md:40]:
```suggestion
  Orchestrate multiple Workflows to build complex Suites for System Testing - [Read More](https://docs.testkube.io/articles/defining-tests).
```

File: README.md [hunk 2]
```diff
@@ -45,54 +29,39 @@ Known vulnerabilities: [![Testkube](https://snyk.io/test/github/kubeshop/testkub
...
+2. **Trigger** - Trigger tests through the API/CLI, from your existing CI/CD/GitOps workflows, using fixed schedules or 
+  by listening to Kubernetes Events or creating execution CRDs - [Read More](https://docs.testkube.io/articles/triggering-tests)
```
Comment (jfermi) [inline README.md:42]:
```suggestion
  by listening to Kubernetes Events or creating execution CRDs - [Read More](https://docs.testkube.io/articles/triggering-tests).
```

File: README.md [hunk 3]
```diff
@@ -45,54 +29,39 @@ Known vulnerabilities: [![Testkube](https://snyk.io/test/github/kubeshop/testkub
...
 
-## Getting Started
+### Documentation
 
-Checkout the [Getting Started](https://docs.testkube.io/articles/getting-started-overview) guides to set up Testkube and run your first tests!
-
-# Documentation
-
-Is available at [docs.testkube.io](https://docs.testkube.io)
+Extensive documentation is available at [docs.testkube.io](https://docs.testkube.io)
```
Comment (jfermi) [inline README.md:64]:
```suggestion
Extensive documentation is available at [docs.testkube.io](https://docs.testkube.io).
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Review by keevans37 — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -216,6 +216,7 @@ version-bump-dev:
 	go run cmd/tools/main.go bump --dev
 
 commands-reference:
+	mkdir -p docs/docs/cli
```
Comment (rangoo94) [inline Makefile:219]: Should it be `docs/docs/cli`, or we should just change `docs.go` to a different directory, like `docs/cli` directly?

**Review by rangoo94 — APPROVED** [review]

File: Makefile (hunk 4 above)
Comment (olensmar) [reply Makefile:219]: TBH idk - since these are temporary they should perhaps go to some `/gen` or `/build` directory? what's the go convention for this?

**Review by exu — APPROVED** [review]

**Comment by olensmar** [comment]
@exu @rangoo94 i can't see how the failed test relates to the changes I made... ok to merge anyway!?

**Comment by rangoo94** [comment]
> @exu @rangoo94 i can't see how the failed test relates to the changes I made... ok to merge anyway!?

@olensmar, sure, we've got flaky tests, and this one is one of the most frequent 👍

**Comment by olensmar** [comment]
thanks @rangoo94 - does that mean I can merge? or should I wait for a resolution?

**Comment by rangoo94** [comment]
> thanks @rangoo94 - does that mean I can merge? or should I wait for a resolution?

Generally you can, but seems that we are missing @testkube-docs review for merging

**Pull request merged**